dist/ramda.js

```diff
@@ -990,8 +990,7 @@
     });
 
     /**
-     * Returns `true` if all elements of the list match the predicate, `false` if
-     * there are any that don't.
+     * Returns `false` if any element does not satisfy the predicate, `true` otherwise.
```

Comment:
@davidchambers: Are you saying that the original *is* the definition of `all` or that the suggested version is?